### DISCUSSION TITLE
Why `9999px` instead of `100%`?

### DISCUSSION THREAD

**Discussion by mjoork** [tailwindlabs/tailwindcss#3373] [Help] [answered]
I am just curious why `rounded-l-full` for example uses `9999px` instead of `100%`? Is there something bad happening if `border-radius` is set to `100%`?

**Comment by adamwathan** [comment, marked as answer]
Hey! Yeah it creates weird elliptical shapes for rectangular shapes instead of pills. Check it out:

https://play.tailwindcss.com/RZDjO7s7Oi

**Reply by mjoork** [reply]
I see, thank you!

**Reply by deadcoder0904** [reply]
was gonna ask the same so glad i used the search. i think this should be mentioned in the docs if it hasn't already :)
